modules/module_loader/include/ModulesApp.h

```diff
@@ -35,6 +35,9 @@
 #ifdef HEAT_CONDUCTION_ENABLED
 #include "HeatConductionApp.h"
 #endif
+#ifdef HEAT_TRANSFER_ENABLED
+#include "HeatTransferApp.h"
+#endif
```

Comment:
So is this part of the 5 step plan having these both together?

Reply:
the 3 lines above those will be removed